client/src/contexts/Shared.styled.js

```diff
@@ -0,0 +1,87 @@
+import styled from 'styled-components';
+
+// Shared by Rating and Q&A
+const Button = styled.button`
+  border: 2px solid;
+  text-align: center;
+  padding: 1.3rem 1rem 1.3rem 1rem;
+  font-size: medium;
+  font-weight: 700;
+  color: ${({ theme }) => theme.colors.secondary};
+  &:hover {
+    background-color:${({ theme }) => theme.colors.tertiary};
+    box-shadow: 0.3rem 0.3rem 0.3rem ${({ theme }) => theme.colors.secondary}
+  }
+`;
+
+const MiniButton = styled(Button)`
+  padding: 0.4rem 0.4rem 0.4rem 0.4rem;
+  font-size: small;
+  font-weight: 400;
+  border-radius: 0.5rem;
+  margin-top: 0.5rem;
+`;
+const ClickableText = styled.button`
+  background-color: transparent;
+  border: none;
+  padding: 0!important;
+  /*optional*/
+  font-family: arial, sans-serif;
+  /*input has OS specific font-family*/
+  color: #069;
+  text-decoration: underline;
+  cursor: pointer;
+`;
+
+const ModalParent = styled.div`
+  position: fixed;
+  background-color: ${({ theme }) => theme.colors.modalBackground};
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 9996;
+  display: ${({ showModal }) => (showModal ? 'block' : 'none')};
+  opacity: ${({ showModal }) => (showModal ? 1 : 0)};
+  pointer-events: disabled;
+  -webkit-transition: all 0.3s;
```

Comment:
I want to look into seeing if babel/ webpack takes care of adding the vendor prefixes for us. there HAS to be a way so that we don't have to write `-webkit-` and stuff

Reply:
That would be super helpful.